src/ClassicUO.Assets/MapLoader.cs

```diff
@@ -609,36 +609,36 @@ public void SanitizeMapIndex(ref int map)
     }
 
     [StructLayout(LayoutKind.Sequential, Pack = 1)]
-    public ref struct StaticsBlock
```

Comment:
Generally if something is a ref struct, changing it to a readonly struct may cause performance regressions.
Ref structs are preferable if the struct will only exist on the stack and/or a reference of it will be passed so that it is not actually copied.